rotkehlchen/tests/unit/test_globaldb_upgrades.py

```diff
@@ -287,3 +287,25 @@ def test_unfinished_upgrades(globaldb: GlobalDBHandler):
     assert globaldb.get_setting_value('ongoing_upgrade_from_version', -1) == -1
     with globaldb.conn.read_ctx() as cursor:
         assert cursor.execute('SELECT value FROM settings WHERE name="is_backup"').fetchone()[0] == 'Yes'  # noqa: E501
+
+
+@pytest.mark.parametrize('globaldb_upgrades', [[]])
+@pytest.mark.parametrize('globaldb_version', [2])
+@pytest.mark.parametrize('target_globaldb_version', [2])
+@pytest.mark.parametrize('reload_user_assets', [False])
+def test_applying_all_upgrade(globaldb):
```

Comment:
we also need in between tests. To test the change you made we need to see that v2->v3 stops at v3 and does not end up putting v4, which is what the previous logic was doing. Can you make sure this is tested?

Reply:
yes, I've added checks to ensure that the version is properly set